airdrop/application/handlers/airdrop_handlers.py

```diff
@@ -64,10 +64,9 @@ def user_eligibility(event, context):
 
 
 @exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger)
-def airdrop_window_claims(event, context):
-    logger.info(f"Got Airdrops Window Claims Events {event}")
-    status, response = AirdropServices(
-    ).airdrop_window_claims(request(event))
+def occam_airdrop_window_claim(event, context):
```

Comment:
Ideally this should have been part of the dynamic class

Reply:
Yes you are right. Occam API's are different and it is clearly mentioned in comments that delete the code once Occam is done.
So I didn't created dynamic class for Occam.